tests/integration_tests/security/test_jail.py

```diff
@@ -79,3 +80,15 @@ def test_default_chroot_hierarchy(test_microvm_with_initrd):
                 CHAR_STATS, test_microvm.jailer.uid, test_microvm.jailer.gid)
     check_stats(os.path.join(test_microvm.jailer.chroot_path(),
                              "firecracker"), FILE_STATS, 0, 0)
+
+
+def test_arbitrary_usocket_location(test_microvm_with_initrd):
+    """Test arbitrary location scenario for the api socket."""
+    test_microvm = test_microvm_with_initrd
+    test_microvm.jailer.extra_args = {'api-sock': 'api.socket'}
```

Comment:
Can we use a `tempfile` path instead of `api.socket`?

Reply:
Hm it is a bit tricky since we would need to create the tempfile inside the chroot but the chroot is not created only after the jailer was spawned...

Reply:
OK, not worth the extra complexity, `api.socket` will do